pokedex/db/load.py

```diff
@@ -389,7 +392,7 @@ def dump(session, tables=[], directory=None, verbose=False, langs=None):
     """
 
     # First take care of verbosity
-    print_start, print_status, print_done = _get_verbose_prints(verbose)
+    print_start, print_done = _get_verbose_prints(verbose)
```

Comment:
This doesn't seem right. Did you test this?

Reply:
Could you change it back? I can't merge this as-is.

Reply:
Should be good to go, sorry about the wait there.